### ISSUE TITLE
No xp points for finishing tests (quiz) on the learning platform

### ISSUE THREAD

**Issue by pawelnowakGIT** [coders-school/site#90]
System doesn't give points for finishing tests (quiz) so it's impossible to achieve TESTER achievement although I've finished at least 5 tests today. Also no sign of it in points log.

![tester quiz](https://user-images.githubusercontent.com/65122804/84907932-1d46c300-b0b4-11ea-8d85-6c9be6a1bd33.JPG)
![tester points](https://user-images.githubusercontent.com/65122804/84907935-1fa91d00-b0b4-11ea-8c68-1c0d2c9dff67.JPG)
![tester osiągnięcie](https://user-images.githubusercontent.com/65122804/84907951-233ca400-b0b4-11ea-9f68-90b8f0aa4282.JPG)

**Comment by ziobron** [comment]
That's true, but for now, we must live with this. I have raised an issue to the author of the plugin of the integration with our platform, but "it works for him". I have also checked if there are any conflicting plugins, but, I have not found any suspects. Maybe later I will be able to take another look. Thanks for raising this Issue.
